lib/tasks/user_stats.rake

```diff
@@ -0,0 +1,10 @@
+task user_stats: 'kitsu:user_stats_setup'
+
+namespace :kitsu do
+  task :user_stats do
```

Comment:
You probably want to use the `environment` dependency so that all of Rails will load for this (otherwise it probably won't be able to access any Rails models)